frontend/packages/console-dynamic-plugin-sdk/src/utils/k8s/k8s-resource.ts

```diff
@@ -77,7 +77,7 @@ type OptionsGet = BaseOptions & {
   requestInit?: RequestInit;
 };
 
-type K8sGetResource = (options: OptionsGet) => Promise<Response>;
+type K8sGetResource = <R extends K8sResourceCommon>(options: OptionsGet) => Promise<R>;
```

Comment:
Didn't have it, didn't really need it... but this allows proper typing for the Get call. Instead of just getting back a `K8sResourceCommon`